Software/Python/grove_mini_motor_driver/grove_mini_motor_driver.py

```diff
@@ -0,0 +1,246 @@
+        self.__writeMotor(self.right_motor, self.right_motor.REVERSE, "right motor warning", speed)
+
+    # command the left motor to go in one of the set directions at a certain speed
+    # direction = {'FORWARD', 'REVERSE'}
+    # speed = 0-100%
+    def setLeftMotor(self, direction, speed):
+        if direction == "FORWARD":
+            self.__print(getTime(), "left motor", "speed = " + str(speed) + "%")
+            self.__writeMotor(self.left_motor, self.left_motor.FORWARD, "left motor warning", speed)
+
+        elif direction == "REVERSE":
+            self.__print(getTime(), "left motor", "speed = " + str(speed) + "%")
+            self.__writeMotor(self.left_motor, self.left_motor.REVERSE, "left motor warning", speed)
+
+    # command the right motor to go in one of the set directions at a certain speed
+    # direction = {'FORWARD', 'REVERSE'}
+    # speed = 0-100%
+    def setRightMotor(self, direction, speed):
+        if direction == "FORWARD":
+            self.__print(getTime(), "right motor", "speed = " + str(speed) + "%")
+            self.__writeMotor(self.right_motor, self.right_motor.FORWARD, "right motor warning", speed)
+
+        elif direction == "REVERSE":
+            self.__print(getTime(), "right motor", "speed = " + str(speed) + "%")
+            self.__writeMotor(self.right_motor, self.right_motor.REVERSE, "right motor warning", speed)
+
+    # command which forces the left motor to stop ASAP
+    # it uses counter acts with an electromotive force in order to stop the motor from spinning faster
+    # might raise some warnings depending on how good the power supply is
+    def stopLeftMotor(self):
+        self.__print(getTime(), "left motor", "stop")
+        self.__writeMotor(self.left_motor, self.left_motor.BRAKE, "left motor warning")
+
+    # command which forces the right motor to stop ASAP
+    # it uses counter acts with an electromotive force in order to stop the motor from spinning faster
+    # might raise some warnings depending on how good the power supply is
+    def stopRightMotor(self):
+        self.__print(getTime(), "right motor", "stop")
+        self.__writeMotor(self.right_motor, self.right_motor.BRAKE, "right motor warning")
+
+    # command which forces both the motor to stop ASAP
+    # it uses counter acts with an electromotive force in order to stop the motors from spinning faster
+    # might raise some warnings depending on how good the power supply is
+    def stopMotors(self):
+        self.stopLeftMotor()
+        self.stopRightMotor()
+
+    # command which kills the power to the motors
+    def disableMotors(self):
+        self.__print(getTime(), "standby motors")
+
+        self.__writeMotor(self.left_motor, self.left_motor.STANDBY, "left motor warning")
+        self.__writeMotor(self.right_motor, self.right_motor.STANDBY, "right motor warning")
+
+"""
+Will be implemented on Monday
```

Comment:
There will be new functionalities the next week. This will make the life simpler for the guy who wants to use this library.